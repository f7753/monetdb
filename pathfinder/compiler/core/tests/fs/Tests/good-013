#!/bin/bash

pf -s9 <<'EOT'
for $a in (1,2,3) return 42
EOT
echo $?
