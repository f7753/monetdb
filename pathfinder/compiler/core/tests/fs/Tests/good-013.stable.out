stdout of test 'good-013` in directory 'compiler/core/tests/fs` itself:


# 15:45:58 >  
# 15:45:58 >  Mtimeout -timeout 60 ./good-013 good-013 
# 15:45:58 >  

0

# 15:45:58 >  
# 15:45:58 >  Done.
# 15:45:58 >  

